src/Angor/Avalonia/Angor.Contexts.Funding/Projects/Application/Dtos/ProjectExtensions.cs

```diff
@@ -1,25 +0,0 @@
-namespace Angor.Contexts.Funding.Projects.Application.Dtos;
-
-// TODO: Check if the logic of these methods is correct
-public static class ProjectExtensions
-{
-    public static long Raised(this ProjectDto project)
-    {
-        return project.Stages.Sum(stage => stage.Amount);
```

Comment:
I am not sure the project dto holds the data for actual raised transactions, only the amount the stage will get in total

Reply:
That file is deleted. The new one is namespace Angor.UI.Model.FullProjectExtensions.